requirements.txt

```diff
@@ -3,6 +3,7 @@ flake8
 # rst_include tool as GitHub doesn't support `.. include::` when rendering
 # previews.
 rst_include
+git+git://github.com/antmicro/symbolator.git@master#egg=symbolator
```

Comment:
Please use the symbolator from SymbiFlow repo.

Reply:
This would require adding the `--output-as-filename` flag introduced in https://github.com/SymbiFlow/symbolator/pull/5 PR.

Reply:
I fixed this requirement